ml-agents/mlagents/trainers/behavior_id_utils.py

```diff
@@ -1,36 +1,46 @@
-from typing import Dict, NamedTuple
+from typing import NamedTuple
+from urllib.parse import urlparse, parse_qs
 
 
 class BehaviorIdentifiers(NamedTuple):
-    name_behavior_id: str
+    """
+    BehaviorIdentifiers is a named tuple if the identifiers that uniquely distinguish
+    an agent encountered in the trainer_controller. The named tuple consists of the
+    fully qualified behavior name, the name of the brain name (corresponds to trainer
+    in the trainer controller) and the team id.  In the future, this can be extended
+    to support further identifiers.
+    """
+
+    behavior_id: str
     brain_name: str
```

Comment:
we should call this behavior_name not brain_name (to match C#). What's behavior_id?

Reply:
I'll do this in a separate PR and also change the use of brain_name/behavior_id/name_behavior_id  everywhere to behavior_name/fully_qualified_behavior_name